tests/src/test/scala/cats/tests/ParallelSuite.scala

```diff
@@ -17,7 +16,7 @@ import scala.collection.immutable.SortedSet
 class ParallelSuite extends CatsSuite with ApplicativeErrorForEitherTest {
 
```

Comment:
can we test the parallel laws?

https://github.com/typelevel/cats/blob/master/laws/src/main/scala/cats/laws/ParallelLaws.scala

Reply:
Will do. 👍